lib/valid_email2/email_validator.rb

```diff
@@ -5,43 +5,44 @@
 module ValidEmail2
   class EmailValidator < ActiveModel::EachValidator
     def default_options
-      { regex: true, disposable: false, mx: false, disallow_subaddressing: false }
+      { regex: true, disposable: false, mx: false, disallow_subaddressing: false, multiple: false }
     end
 
     def validate_each(record, attribute, value)
       return unless value.present?
       options = default_options.merge(self.options)
 
-      address = ValidEmail2::Address.new(value)
+      value_spitted = options[:multiple] ? value.split(',').map(&:strip) : [value]
+      addresses = value_spitted.map { |v| ValidEmail2::Address.new(v) }
 
-      error(record, attribute) && return unless address.valid?
+      error(record, attribute) && return unless addresses.all?(&:valid?)
 
       if options[:disallow_dotted]
-        error(record, attribute) && return if address.dotted?
+        error(record, attribute) && return if addresses.all?(&:dotted?)
```

Comment:
I'm pretty sure we should use `any?` here instead, as this will only fail if all the addresses are dotted.

The same applies for every error chuck under this that you've changed

Reply:
Your are right. Only the checks with `unless` have to got the `all?` method. I updated the pull request.